src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml

```diff
@@ -7,9 +7,9 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:fabric="clr-namespace:AccessibilityInsights.CommonUxComponents.Controls;assembly=AccessibilityInsights.CommonUxComponents"
```

Comment:
>fabric [](start = 10, length = 6)

I changed `controls` to `fabric` to be consistent with other places where this namespace was referenced, and to reserve `controls` for `AccessibilityInsights.SharedUx.Controls`